reconcile/utils/oc_map.py

```diff
@@ -61,30 +60,22 @@ def __init__(
         self._lock = Lock()
         self._jh_ports: dict[str, int] = {}
 
-        # init a namespace with clusterAdmin with both auth tokens
-        # OC_Map is used in various places and even when a namespace
-        # declares clusterAdmin token usage, many of those places are
-        # happy with regular dedicated-admin and will request a cluster
-        # with oc_map.get(cluster) without specifying privileged access
-        # specifically
-        privileged_clusters: list[OCConnectionParameters] = [
-            c for c in connection_parameters if (c.is_cluster_admin or cluster_admin)
-        ]
-        unprivileged_clusters: list[OCConnectionParameters] = [
-            c
-            for c in connection_parameters
-            if not (c.is_cluster_admin or cluster_admin)
-        ]
+        unprivileged_connections = (
+            c for c in connection_parameters if not c.is_cluster_admin
+        )
+        privileged_connections = (
+            c for c in connection_parameters if c.is_cluster_admin
+        )
```

Comment:
Is not this behaving the same way as before? Where the non-privileged connection is created when `c.is_cluster_admin` is set?

Reply:
Yeah the logic that fixes the bug is moved to oc_connection_parameters: https://github.com/app-sre/qontract-reconcile/pull/3249/files#diff-d532c0b963df575b717e917a3238abec2f0e7a38ee9aff9efb717221bb533f0bR226-R236

Reply:
As discussed offline, I removed redundant filtering block. Thanks for the suggestion :bow: